routes/rateLimiter.js

```diff
@@ -0,0 +1,16 @@
+const rateLimit = require("express-rate-limit");
+
+const webcamLimiter = rateLimit({
+    windowMs: 60 * 60 * 1000, // 30 min window
+    max: 10, // start blocking after 5 requests
+    message: {errorMsg: "Too many requests from this IP, please try again after an hour"}
```

Comment:
Maybe change the error message to something more friendly?
ex. "Too many requests, please try again after an hour"

Reply:
sure